src/test/java/ai/elimu/util/ml/ReadingLevelUtilTest.java

```diff
@@ -0,0 +1,59 @@
+package ai.elimu.util.ml;
+
+import ai.elimu.model.v2.enums.ReadingLevel;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class ReadingLevelUtilTest {
+
+    @Test
+    public void testPredictReadingLevel_Level1() {
+
+        String modelFilePath = "src/test/resources/ai/elimu/util/reading_level/model1.pmml";
```

Comment:
@Aryant-Tripathi Here, I suggest you load the original model file under `src/main/resources` instead of making a copy. This is because the original model file will be regularly updated, and we want to unit tests to be as close as possible to changing real-world scenarios.

Reply:
@jo-elimu Will it be ok to use the same file for controller (prod) environment and for test case environment. I thought we should differentiate the model , that's why I have used  a sample model file. Do I need to remove this and use the existing one.

Reply:
@Aryant-Tripathi Yes, we use the same `.pmml` model file everywhere (across `DEV`/`PROD` environments, and across `ENG`/`HIN`/`TGL` languages), so it's okay to use it for the unit tests.